### DISCUSSION TITLE
What are the exact names for the validation metrics for the config file?

### DISCUSSION THREAD

**Discussion by darylsuen** [neuralhydrology/neuralhydrology#179] [Q&A]
Hello! 
I'm very new to the library. I'm looking for the exact names of the metrics including case-sensitivity for entering into the config file. The metrics documentation describes the metrics well but doesn't say how the name of each metric should be entered.
Kind regards.

**Comment by gauchm** [comment]
That's a fair point, we should probably add these names to the documentation. For now, you can find all the metric names [in metrics.py](https://github.com/neuralhydrology/neuralhydrology/blob/e4329c32b73d2ea9a0490aaef822ded4a6f5d530/neuralhydrology/evaluation/metrics.py#L842-L870).
